assets/js/script.js

```diff
@@ -150,48 +168,65 @@ function render() {
   $('.js-tab').html('')
   $('.js-preview').html('')
   $('.js-tab').removeClass('active')
+  $('.js-tab').parent().removeClass('active')
   $('.js-swatch').removeClass('active')
   $('.js-loading').hide()
 
   // render bag color
-  $('.js-bag').css({backgroundImage: "url("+(window.baseUrl || '')+"assets/img/bags/danny-"+state.bag+".png)"})
 
-  // render letters in bag and preview
+  $('.js-bag').css({ backgroundImage: "url(" + (window.baseUrl || '') + "assets/img/bags/danny-" + state.bag + ".png)" })
+
+  $('.js-tab').each(function() {
+      if ($(this).data('index') != -1) {
+        var ind = $(this).data('index')
+        var src = `${window.baseUrl || ''}assets/img/letters/` + state.letters[ind].toUpperCase() + `-` + state.materials[ind] + `.png`;
+        $(this).css({ background: "url(" + (window.baseUrl || '') + src + ") no-repeat", backgroundSize: "contain", backgroundPosition: "center" })
+
+      } else {
+        $(this).html('All')
+      }
+    })
+    // render letters in bag and preview
   state.letters.split('').forEach(function(l, i) {
-    var letter = Letter({ letter: l, material: state.materials[i], index: i})
+    var letter = Letter({ letter: l, material: state.materials[i], index: i })
     $('.js-loading').show()
     $('.js-letters').append(letter)
     letter[0].onload = function() {
       $('.js-loading').hide()
-      if( state.activeLetter === i ) {
+      if (state.activeLetter === i) {
         $('.js-preview').append(letter.clone())
       }
     }
   })
 
   // show editor palette
-  if( state.editing ) {
+
+  /*
+  if (state.editing) {
     $('.js-palette').show().removeClass('closed')
 
-    if( state.editingBag ) {
+    if (state.editingBag) {
       $('.js-bag-editor').show()
       $('.js-letter-editor').hide()
     } else {
       $('.js-letter-editor').show()
       $('.js-bag-editor').hide()
     }
-  } else if( lastState.editing ) {
+  } else if (lastState.editing) {
     $('.js-palette').addClass('closed')
     $('input').blur()
   }
+  */
 
   // select input if we're showing a new input
-  if( state.editing !== lastState.editing || (state.activeLetter !== lastState.activeLetter && state.activeLetter === state.letters.length) ) {
+  if (state.editing !== lastState.editing || (state.activeLetter !== lastState.activeLetter && state.activeLetter === state.letters.length)) {
     $('.js-input').select()
   }
 
   // render active tab
+
   $(`.js-tab[data-index=${state.activeLetter}]`).addClass('active')
+  $(`.js-tab[data-index=${state.activeLetter}]`).parent().addClass('active')
```

Comment:
so this is gonna be fragile bc of the `parent` selector, you can make it more resilient in case other dom elements are added between this element and the parent by doing:

```js
$(`.js-tab[data-index=${state.activeLetter}]`).parents('.js-back-tab').addClass('active')
```